components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -771,48 +815,13 @@ void RRTMGPRadiation::run_impl (const double dt) {
       // n2 and co are set to constants and are not handled by trcmix;
       // the rest are handled by trcmix
       real2d tmp2d = subview_2d(m_buffer.tmp2d);
-      const auto gas_mol_weights = m_gas_mol_weights;
       for (int igas = 0; igas < m_ngas; igas++) {
         auto name = m_gas_names[igas];
 
         // We read o3 in as a vmr already
         if (name=="o3") continue;
 
         auto d_vmr = get_field_out(name + "_volume_mix_ratio").get_view<Real**>();
-        if (name == "h2o") {
-          // h2o is (wet) mass mixing ratio in FM, otherwise known as "qv", which we've already read in above
-          // Convert to vmr
-          const auto policy = ekat::ExeSpaceUtils<ExeSpace>::get_default_team_policy(ncol, m_nlay);
-          Kokkos::parallel_for(policy, KOKKOS_LAMBDA(const MemberType& team) {
```

Comment:
The calculation of h2o was fine, since it only computed the value for the cols in the current chunk. However, for the other gases, we were recomputing all the `m_ncol` entries for every chunk.